airbyte-cdk/python/unit_tests/sources/streams/test_stream_read.py

```diff
@@ -232,9 +241,10 @@ def test_full_refresh_read_a_single_slice(constructor):
     [
         pytest.param(_stream, id="synchronous_reader"),
         pytest.param(_concurrent_stream, id="concurrent_reader"),
+        pytest.param(_stream_with_no_cursor_field, id="no_cursor_field"),
```

Comment:
nit: it might be worth splitting into its own test since the goal isn't so much to verify the behavior when reading two slices, but specifically to verify the get_updated_state method isn't called on a stream that doesn't have a cursor_field.